src/schema/schema.rs

```diff
@@ -86,6 +86,8 @@ impl TransitionType {
 
 impl TransitionType {
     pub const BLANK: Self = TransitionType(u16::MAX);
+    /// Easily check if the TransitionType is blank with convention method
+    pub fn is_empty(self) -> bool { self.0 == u16::MAX }
```

Comment:
```suggestion
    pub fn is_blank(self) -> bool { self.0 == u16::MAX }
```

It is not empty: transition has information and data. It is "blank" meaning "it covers transfer of all the state to a new seals, 1-to-1". Like "blank trait implementations" in rust cover all types and make them implementing certain trait. This is quite far from the concept of "empty"!

Reply:
Also, I do not like that we do guess the constant value. It should be
```suggestion
    pub fn is_empty(self) -> bool { self == Self::BLANK }
```